creme/creme_core/views/testjs.py

```diff
@@ -102,11 +102,7 @@ def __init__(self, name, user, image_url):
         self.duration = mark_safe(print_duration(
             self, f'{randint(0, 23)}:{randint(0, 59)}:{randint(0, 59)}', user, None
         ))
-        self.foreignkey = (
-            None
-            if property is None else
-            mark_safe(print_foreignkey_html(self, property, user, None))
-        )
+        self.foreignkey = mark_safe(print_foreignkey_html(self, property, user, None))
```

Comment:
It's the built-in "property" ? :thinking:

Reply:
:facepalm: I DO need holidays